examples/configuration/echo.lua

```diff
@@ -0,0 +1,7 @@
+local policy_chain = require('apicast.policy_chain').build({ })
+
+policy_chain:insert(policy_chain.load('apicast.policy.echo'))
```

Comment:
This insert + load combination looks a bit confusing to me. I wonder if we should make `.load` private.

Reply:
And just rely on `require(...).new()` instead of `.load` ?

Reply:
What about `.insert` calling `.load` like `.build` does and making `.load` private?

I think that `policy_chain:insert('apicast.policy.echo')` or `policy_chain:insert(echo.new(something))` looks better than `policy_chain:insert(policy_chain.load(...))`.

Reply:
I'd like `insert` to be an API that does one thing only. So I'd prefer it does not initialize the module because then it would also accept the configuration.

I agree `.load` is kind of an useless abstraction and it would be better to use plain require. The only reason for is existence was that I wanted to have an abstraction so the internals can be replaced without much hassle. 
But when replacing internal and for example changing `:new` to `.new` the policies would have to change anyway and it would leak this concept to the policy. But at least all the uses of the policy would not have to change.

So we should agree if we want to expose the policy initialization to users and deal with the consequences if we need to ever change it or expose some method to load/initialize policies.

Reply:
The new example looks like https://github.com/3scale/apicast/pull/496/commits/02ad4ab34e95041f9b288c73b27565d845aae123 so it uses plain `require`. Guess that is ok for now.